template/deployments/helm/charts/values.yaml

```diff
@@ -60,3 +59,21 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+generatorlabel: draft
+
+serviceport: 80
+
+chartLabel: "{{ printf \"%s-%s\" .Chart.Name .Chart.Version | replace \"+\" \"_\" | trunc 63 | trimSuffix \"-\" }}"
+
+commonLabels:
+  helm.sh/chart: "{{ .Values.chartLabel }}"
```

Comment:
these values may be used literally in the templates instead of being substituted